app/serializers/concerns/user_sidebar_mixin.rb

```diff
@@ -45,6 +45,7 @@ def include_sidebar_list_destination?
   private
 
   def sidebar_navigation_menu?
-    !SiteSetting.legacy_navigation_menu?
+    @sidebar_navigation_menu ||=
+      !SiteSetting.legacy_navigation_menu? || options[:enable_sidebar_param] == "1"
```

Comment:
I'm not sure we need the memoization, especially since the value is a boolean, and will be re-evaluated anyway if the first evaluation comes out as `false`. 🙂

Reply:
oops thank you! good catch